<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>1130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QComboBox" name="cbJoystick">
      <item>
       <property name="text">
        <string>Cursor Interface</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kempston Interface</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sinclair Interface 2 - port 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sinclair Interface 2 - port 2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="cbShowControls">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Show controls</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="ScreenWidget" name="screen" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="cbKeyboardCapture">
      <property name="text">
       <string>Capture Keyboard</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="btnReset">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTabWidget" name="controlsTab">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="keyboard">
       <attribute name="title">
        <string>Keyboard</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="KeyboardWidget" name="keyboardwd" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="joystick">
       <attribute name="title">
        <string>Joystick</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pbRight">
          <property name="font">
           <font>
            <pointsize>50</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>→</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pbDown">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>50</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>↓</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="pbFire">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>70</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>✸</string>
          </property>
          <property name="iconSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pbLeft">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>50</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>←</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pbUp">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>50</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>↑</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_a_snapshot"/>
    <addaction name="actionLoad_a_snapshot_z80"/>
    <addaction name="action_Save_a_snapshot"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_a_screenshot"/>
    <addaction name="actionMake_a_scrennshot"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuMachine">
    <property name="title">
     <string>&amp;Machine</string>
    </property>
    <addaction name="action_Reset"/>
    <addaction name="action_NMI"/>
    <addaction name="separator"/>
    <addaction name="actionSpectrum_48k"/>
    <addaction name="actionSpectrum_128k"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMachine"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_a_snapshot">
   <property name="text">
    <string>&amp;Load a snapshot (.sna)...</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load a snapshot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="action_NMI">
   <property name="text">
    <string>&amp;NMI</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Save_a_snapshot">
   <property name="text">
    <string>&amp;Save a snapshot (.z80)</string>
   </property>
  </action>
  <action name="actionSpectrum_48k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Spectrum 48k</string>
   </property>
  </action>
  <action name="actionSpectrum_128k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spectrum 128k</string>
   </property>
  </action>
  <action name="actionMake_a_scrennshot">
   <property name="text">
    <string>Save a screenshot</string>
   </property>
  </action>
  <action name="actionLoad_a_screenshot">
   <property name="text">
    <string>Load a screenshot...</string>
   </property>
  </action>
  <action name="actionLoad_a_snapshot_z80">
   <property name="text">
    <string>Load a snapshot (.z80)...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenWidget</class>
   <extends>QWidget</extends>
   <header location="global">screenwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KeyboardWidget</class>
   <extends>QWidget</extends>
   <header location="global">keyboardwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
